ietf-optical-impairment-topology.tree

```diff
@@ -58,24 +58,21 @@ module: ietf-optical-impairment-topology
                       +--ro loss?   decimal64
   augment /nw:networks/nw:network/nw:node/tet:te
             /tet:tunnel-termination-point:
-    +--ro OTSiG-element* [OTSiG-identifier]
-    |  +--ro OTSiG-identifier    int16
-    |  +--ro OTSiG-container
-    |     +--ro OTSi* [OTSi-carrier-id]
-    |        +--ro OTSi-carrier-id           int16
-    |        +--ro transponder-ref?          leafref
-    |        +--ro transceiver-ref?          leafref
-    |        +--ro configured-mode?          leafref
-    |        +--ro OTSi-carrier-frequency?   decimal64
-    |        +--ro OTSi-signal-width?        decimal64
-    |        +--ro channel-delta-power?      decimal64
-    |        +--ro central-frequency?        frequency-thz
-    |        +--ro channel-output-power?     dbm-t
-    |        +--ro channel-input-power?      dbm-t
-    |        +--ro total-input-power?        dbm-t
-    +--ro transponder-list* [transponder-id]
-       +--ro transponder-id      uint32
-       +--ro transceiver-list* [transceiver-id]
+    +--ro otsi-group* [otsi-group-id]
+    |  +--ro otsi-group-id    int16
+    |  +--ro otsi* [otsi-carrier-id]
+    |     +--ro otsi-carrier-id           int16
+    |     +--ro transponder-ref?          leafref
+    |     +--ro transceiver-ref?          leafref
+    |     +--ro configured-mode?          leafref
+    |     +--ro OTSi-carrier-frequency?   frequency-thz
+    |     +--ro OTSi-carrier-bandwidth?   frequency-ghz
```

Comment:
OTSi Carrier Bandwidth isn't defined in G.698.2, suggest to use the definition @ggalimba56 provided https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-optical-impairment-topology-yang/pull/36#pullrequestreview-509117992

Reply:
I see you point about OTSi-carrier-bandwidth : last meeting was discussed to move the attribute in the common grouping "common-transceiver-configured-param", and this is what I've done. But no problem if we discover it would not be the case. Let's discuss.

Reply:
solved with 8b7c07ee04c2db56b478ed4bb269308006a4e976